docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md

```diff
@@ -443,14 +449,14 @@ Data Providers will usually follow one of two patterns:
    convenient way to get started with the AAS. Due to the risk of data duplication and unclear initial ingestion
    mechanisms, it may not scale to industrial sizes.
 2. Delegation: Wrapping another API or a database may deploy the Submodel API as a new facade. It delegates the incoming
-   requests to the respective backend systems. This is particularly feasible in the Catena-X dataspace since it
+   requests to the respective backend systems. This is particularly feasible in the Catena-X dataspace.
 
 Offering data to the network requires mappings that are naturally dependent on the data source format. More on data integration
 can be found in the corresponding [CX e.V. guide](https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Onboarding/DataIntegrationPatterns_Guide_Final_V1.pdf).
 
 ### Patterns for DTRs
 
-Usually, a DTR will implement a persistence with the specified AAS-APIs for data ingestion specified in the SSP-001 by
+Usually, a DTR will implement a persistence with the specified AAS-APIs for data ingestion specified in the SSP-0001 by
```

Comment:
I'm pretty sure the profiles only have three-digit ranges.

Reply:
This is true